app/jobs/scheduled/summaries_backfill.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module ::Jobs
+  class SummariesBackfill < ::Jobs::Scheduled
+    every 5.minutes
+
+    def execute(_args)
+      return if !SiteSetting.discourse_ai_enabled
+      return if !SiteSetting.ai_summarization_enabled
+      return if SiteSetting.ai_summary_backfill_maximum_topics_per_hour.zero?
+
+      # Backfill up to 150 summaries per 5-minute run.
+      limit_per_job = 150
```

Comment:
Shouldn't this be `ai_summary_backfill_maximum_topics_per_hour / 12` ?

Reply:
Isn't that worse when max topics per hour is small?

Reply:
Should be the same right? every 5 minutes is 12 calls an hour.